Source/ToolCore/JSBind/cplusplus/Bind.cpp

```diff
@@ -1910,6 +1910,7 @@ bool Bind::visit(SimpleDeclarationAST *ast)
 
             if (Function *funTy = decl->type()->asFunctionType()) {
                 funTy->setMethodKey(methodKey);
+                funTy->setVisibility(_visibility);
```

Comment:
It seems odd that this modification would be necessary, if it is, let's make sure to mark it as modified and the reason for the modification